proposals/serverlessworkflow.md

```diff
@@ -0,0 +1,170 @@
+interoperability specification, as shown by the significant participation
+from key industry companies.
+
+We believe that the CNCF provides the proper home for this due to its
+commitment to the promotion and development of open, vendor-neutral projects.
+Additionally, the wide breadth of the CNCF members will provide the feedback
+necessary to ensure the Serverless Workflow specification isn't too limited in its
+scope and appeals to as many constituents of the cloud native community
+as possible.
+
+**Sponsor / Advisor from TOC**:
+
+**Preferred maturity level**:
+
+Sandbox
+
+**License**:
+
+Apache License v2.0
+
+**Source control repositories**:
+
+CNCF Serverless WG repository (workflow directory): https://github.com/cncf/wg-serverless/tree/master/workflow
+
+**External dependencie**:
+
+None
+
+**Initial Maintainers**:
+
+We have the
+[GOVERNANCE](https://github.com/cncf/wg-serverless/blob/master/workflow/spec/governance/readme.md)
+doc which explains the processes we follow.
+
+Also we have the:
+[OWNERS](https://github.com/cncf/wg-serverless/blob/master/workflow/spec/governance/owners.md)
+doc lists community members currently responsible for overseeing, reviewing, and approving changes to specification documents.
+
+**Infrastructure Requests**:
+
+We would like to request a dedicated github repository for the Serverless Workflow specification.
+
+We would like to request a dedicated website.
+
+**Communication Channels**:
+
+Mailing list: https://lists.cncf.io/g/cncf-wg-serverless
+
+Slack: #serverless-workflow Slack channel under CNCF's Slack workspace https://slack.cncf.io/
+
+We have monthly zoom calls, first Monday every month @ 10-11am PT:
+https://zoom.us/my/cncfserverlesswg
+
+We track our monthly meeting information here: https://docs.google.com/document/d/1xwcsWQmMiRN24a7o7oy9MstzMroAup31oOkM5Dru1jQ
+
+**Issue tracker**:
+
+Issues are tracked with GitHub Issues: https://github.com/cncf/wg-serverless/issues
+
+Changes are tracked with GitHub PRs: https://github.com/cncf/wg-serverless/pulls
+
+**Website**:
+
+Serverless Workflow does not have its own website.
+
+**Release Methodology and Mechanics**
+
+With the large growth of the specification in the last months, we are formalizing the
+specification roadmap at the moment and working on the v.0.1 release.
+
+Beyond what will be defined there, the sub-group will decide when
+significate progress has been made to warrant a new release.
+
+**Social Media Accounts**:
+
+**Contributor statistics**:
+
+Meeting attendance is tracked [here](https://docs.google.com/document/d/1xwcsWQmMiRN24a7o7oy9MstzMroAup31oOkM5Dru1jQ/edit#heading=h.g2rizfze8av2).
+
+As can been seen in that document, Serverless Workflow monthly calls have regular
+attendance.
+
+Without implying endorsement, the following companies have attended at least
+one meeting:
+
+Nokia, Red Hat, Camunda, Futurewei, Pivotal / VMware, IBM, Twilio, Codid
+
+And the following have voting rights (today), which means they regularly
```

Comment:
Does this apply to workflow? I thought you guys had "maintainers" and those are the folks who can "vote".  I didn't think your voting rights were based on attendance like CE does it

Reply:
you are correct, updated